Jellyfin.Api/Controllers/AudioController.cs

```diff
@@ -138,7 +139,8 @@ public AudioController(AudioHelper audioHelper)
         [FromQuery] int? audioStreamIndex,
         [FromQuery] int? videoStreamIndex,
         [FromQuery] EncodingContext? context,
-        [FromQuery] Dictionary<string, string>? streamOptions)
+        [FromQuery] Dictionary<string, string>? streamOptions,
+        [FromQuery] bool? enableAudioVbrEncoding)
```

Comment:
```suggestion
        [FromQuery] bool enableAudioVbrEncoding = true)
```

I would specify the default value here so it gets pulled into the api spec